src/serialized-utilities.cpp

```diff
@@ -0,0 +1,160 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+#include "serialized-utilities.h"
+#include <device.h>
+
+using json = nlohmann::json;
+
+namespace librealsense {
+namespace serialized_utilities {
+
+static const char* SCHEMA_VERSION = "1.0";
+
+json_preset_reader::json_preset_reader( const std::string & json_content ) : _parameters(nullptr)
+{
+    _root = json::parse( json_content );
+    if (validate_schema())
```

Comment:
Seems redundant to `validate` then set the schema version...
Why not just call it `init_schema` and have it set `_schema_version`?

Reply:
👍